checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/session/request/handlers/VerifiedTokensSessionRequestHandler.kt

```diff
@@ -17,17 +17,34 @@ internal class VerifiedTokensSessionRequestHandler(
     private val sessionRequestHandlerConfig: SessionRequestHandlerConfig
 ) : SessionRequestHandler {
 
+    /**
```

Comment:
I would just add a class doc here, instead of documentation on each function.

Reply:
updated